Agents/AccessAgent/README.md

```diff
@@ -1,163 +1,152 @@
+
+- `label`
+- `queryEndpoint`
+- `updateEndpoint`
+
+Once the file is populated, run the bash script `./access-agent-dev-stack/uploadRouting.sh` using the `bash ./uploadRouting.sh` command in the `./access-agent-dev-stack` directory. Adjust the `accessagentURL` field if running within a stack. Note that if a "label" already exists, the uploader will not overwrite any information. Instead, please manually modify the triples on the namespace.
+
+> RDB Access Agent
+
+1.  In the knowledge graph, create a new namespace based on the specified `RDB_STOREROUTER_ENDPOINT` environment variable (default is `ontordbrouter`) to store the routing information. If you wish to use another namespace, please adjust your Docker environment variables accordingly to function in the stack or standalone containers.
+2.  For each PostgreSQL database to be accessed, 4 triples need to be added to the namespace in (1). Example triples for accessing a database `test` is shown below. Note to replace all occurrences of `test` with your required database name to ensure the label for the target resource works.
+
 ```
-docker push ghcr.io/cambridge-cares/access-agent:X.Y.Z
+// Database url
+<http://host.docker.internal:9999/blazegraph/ontordbrouter/test> <http://www.theworldavatar.com/kg/ontordbrouter/hasUrl> "jdbc:postgresql://localhost:5432/test".
+
+// Classes
+<http://host.docker.internal:9999/blazegraph/ontordbrouter/test> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.theworldavatar.com/kg/ontordbrouter/TargetRDBResource>.
+<http://host.docker.internal:9999/blazegraph/ontordbrouter/test> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2002/07/owl#NamedIndividual>.
+
+// Label used as an identifier for accessing the database
+<http://host.docker.internal:9999/blazegraph/ontordbrouter/test> <http://www.w3.org/2000/01/rdf-schema#label> "test".
 ```
-where X.Y.Z is the new version number.
 
-### Integration tests
-Once built, the AccessAgentIntegrationTests and RDBAccessAgentIntegrationTests can be run on the new version of the AccessAgent 
-by updating the ACCESS_AGENT_VERSION variable in the AccessAgentIntegrationTest class and the RDBAccessAgentIntegrationTest class
-before running the tests.
+### 3.2 Calling the Access Agent
+
+The access agent is typically intended for use by other agents, rather than being called on its own to limit access to the underlying data in the knowledge graph. Accordingly, this document does not describe the available routes. Instead, developers can call the access agent in two ways:
```

Comment:
I'm really confused by "rather than being called on its own to limit access to the underlying data in the knowledge graph". Restricting access is one of the possible applications of the Access Agent, but that isn't how it is being used at present. Its primary purpose is to _enable_ access (in a way that doesn't require you to remember/hard-code awkward URLs that may change over time...).

I also don't understand what you mean by "this document does not describe the available routes". Isn't it the purpose of this document to describe the available routes?

Reply:
These sentences have been removed as they are more confusing and does not convey the message of using the access agent in the following two manners.